src/app/LayoutCommon.astro

```diff
@@ -3,14 +3,31 @@ import pkg from '../../package.json' assert { type: 'json' };
 import { getFromCookies } from '../api/Auth';
 import { SITE_TITLE } from './constants.js';
 import { envConfig } from '../config/envConfig';
+import { excludeKeys } from '../config/CommonConstant';
 
 const { class: clazz, metaData } = Astro.props;
 
-const initData: any = {};
+const allEnvKeys = [...Object.keys(process.env), ...Object.keys(import.meta.env)];
 
-const envKeys = [...Object.keys(process.env), ...Object.keys(import.meta.env)];
-envKeys.forEach((item) => {
-	initData[item] = process.env[item] || import.meta.env[item];
+if (allEnvKeys.length === 0) {
+  throw new Error('No environment keys were found in process.env or import.meta.env.');
+}
+
+const exposedEnvKeys = allEnvKeys.filter((key) => !excludeKeys.includes(key));
+
+const initData: Record<string, any> = {};
+const excludedEnvData: Record<string, any> = {};
+
+allEnvKeys.forEach((key) => {
+  const value = process.env[key] || import.meta.env[key];
+  
+  if (!value) return; 
+  
+  if (excludeKeys.includes(key)) {
```

Comment:
This condition is not needed, as this const is never used.

Reply:
Yes, I removed this condition, as we don't use this variable anywhere in the layout component.